examples/src/SingleSelectExample.elm

```diff
@@ -63,7 +69,7 @@ view model =
                 )
             ]
         , form [ onSubmit HandleFormSubmission ]
-            [ input [ style "margin-bottom" "1rem" ] []
+            [ input [ style "margin-bottom" "20rem" ] []
```

Comment:
We should give a hint that the select opens above if there's not enough space below. Otherwise this spacing seems a bit weird.